candi/bashible/common-steps/node-group/042_configure_ethtool_tuner.sh.tpl

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Flant JSC
```

Comment:
why "node-group" dir, not "all"?

Reply:
This has to do with setting up the Node. There are already other files related to Node settings like 041_configure_sysctl_tuner.sh.tpl. I think they should be kept together to make it easier to navigate the project.